stable/cluster-autoscaler/templates/clusterrole.yaml

```diff
@@ -104,11 +104,11 @@ rules:
     - get
 {{- if .Values.rbac.pspEnabled }}
   - apiGroups:
-    - extensions
+    - policy
```

Comment:
Does this work on older versions of Kubernetes? Should we branch on version?

Reply:
`extensions` would be right up to v1.13, v1.14 and on would "require" `policy`. It seems the API is kept backwards compatible for now and `extensions` could still be used up to v1.15. We could branch on API version, or just leave it at `extensions` for now.

Reply:
Let's maybe leave it as `extensions` for now. Your call.